tests/tests_suite_test.go

```diff
@@ -186,6 +187,28 @@ func startVM() (context.Context, VM) {
 			}
 			return nil
 		},
+		// Firmware
+		func(m *types.MachineConfig) error {
+			FW := os.Getenv("FIRMWARE")
+			if FW != "" {
+				m.Args = append(m.Args,
+					"-bios", FW)
+			}
+
+			return nil
+		},
+		// UKI boot
+		func(m *types.MachineConfig) error {
+			drive := os.Getenv("UKI_DRIVE")
```

Comment:
Sorry for the late comment. I think the test below can easily turn to false positive if one omits these 2 environment variables. A quick solution would be to check in the test that these 2 environment variables are set (like [we do for `CONTAINER_IMAGE` in the upgrade test](https://github.com/kairos-io/kairos/blob/224d7c0045ddd5c1d648e7f379359f175aaabbd4/tests/upgrade_cli_test.go#L16-L21))